compute/signed-metadata/pom.xml

```diff
@@ -0,0 +1,66 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud.example.jwt</groupId>
+  <artifactId>compute-signed-metadata</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>compute-signed-metadata</name>
+  <dependencies>
```

Comment:
Usually we ask that you reference the parent pom .. so that checkstyle and other plugins run at verify time.

Reply:
Done